src/Form/FieldSelect.js

```diff
@@ -0,0 +1,198 @@
+import classNames from 'classnames/dedupe';
+import React from 'react';
+
+import Dropdown from '../Dropdown/Dropdown';
+
+class FieldSelect extends React.Component {
+
+  handleChange(selectedValue) {
+    this.props.handleEvent('change',
+      this.props.name, selectedValue);
+  }
+
+  hasError() {
+    let {props} = this;
+    let validationError = props.validationError;
+    return !!(validationError && validationError[props.name]);
+  }
+
+  getDropDown(dropdownItems) {
+    let {
+      defaultValue,
+      dropdownMenuButtonClassName,
+      dropdownMenuClassName,
+      dropdownMenuListClassName,
+      dropdownMenuWrapperClassName,
+      persistentID
+    } = this.props;
+
+    if (defaultValue == null) {
+      defaultValue = dropdownItems[0];
+    }
+
+    if (typeof defaultValue === 'string'){
+      defaultValue = {
+        html: defaultValue,
+        id: defaultValue.toLowerCase()
+      };
+    }
+
+    let dropdownMenuButtonClassSet = classNames('button dropdown-toggle',
```

Comment:
This feels repititive, maybe we could do something like this:
```js
getDropDown(dropdownItems) {
  let {defaultValue, persistentID} = this.props;

  let classes = {
    buttonClassName: {
      defaults: 'button dropdown-toggle',
      propsName: 'dropdownMenuButtonClassName'
    },
    dropdownMenuClassName: {
      defaults: 'dropdown-menu',
      propsName: 'dropdownMenuClassName'
    },
    dropdownMenuListClassName: {
      defaults: 'dropdown-menu-list',
      propsName: 'dropdownMenuListClassName'
    },
    wrapperClassName: {
      defaults: 'dropdown',
      propsName: 'dropdownMenuWrapperClassName'
    }
  };

  classes = Object.keys(classes).reduce(function (classSet, className) {
    let {defaults, propsName} = classes[className];
    classSet[className] = classNames(defaults, this.props[propsName]);
  }, classes);

  return (
    <Dropdown
      items={dropdownItems}
      onItemSelection={this.handleChange.bind(this)}
      transition={true}
      persistentID={persistentID}
      initialID={defaultValue.id}
      {...classes} />
  );
}
```

Or if you keep the names the same you could do something like this:
```js
getDropDown(dropdownItems) {
  let {defaultValue, persistentID} = this.props;

  let classes = {
    buttonClassName: 'button dropdown-toggle',
    dropdownMenuClassName: 'dropdown-menu',
    dropdownMenuListClassName: 'dropdown-menu-list',
    wrapperClassName: 'dropdown'
  };

  classes = Object.keys(classes).reduce(function (classSet, className) {
    classSet[className] = classNames(classes[className], this.props[className]);
  }, classes);

  return (
    <Dropdown
      items={dropdownItems}
      onItemSelection={this.handleChange.bind(this)}
      transition={true}
      persistentID={persistentID}
      initialID={defaultValue.id}
      {...classes} />
  );
}

```

Not sure if that is the best approach, but something that would make this less repetitive would be good